src/main/java/org/folio/circulation/resources/foruseatlocation/HoldByBarcodeResource.java

```diff
@@ -162,6 +160,17 @@ private Result<HoldByBarcodeRequest> setActionToHeld(Result<HoldByBarcodeRequest
     return request.map(req -> req.withLoan(req.getLoan().withAction(LoanAction.HELD_FOR_USE_AT_LOCATION)));
   }
 
+  private CompletableFuture<Result<HoldByBarcodeRequest>> updateLoan(HoldByBarcodeRequest request, LoanRepository loanRepository) {
+    log.debug("updateLoan:: loan parameter: {}", request.getLoan().asJson());
+    return loanRepository.updateLoan(request.getLoan())
+      .thenApply(loanResult -> loanResult.map(request::withLoan));
+  }
+
+  private CompletableFuture<Result<HoldByBarcodeRequest>> publishEvent (HoldByBarcodeRequest request, EventPublisher eventPublisher) {
+    return eventPublisher.publishUsageAtLocationEvent(request.getLoan(), LogEventType.LOAN)
```

Comment:
Recently you've added two PubSub event types to the EventType enum:
```
ITEM_HELD_FOR_USE_AT_LOCATION,
ITEM_PICKED_UP_FOR_USE_AT_LOCATION
```
but they don't seem to be used anywhere in the code. If you're only sending circulation log events (there's already a type for those - `EventType.LOG_RECORD`), maybe these two new types need to be removed from the EventType enum?